app/services/container_dashboard_service.rb

```diff
@@ -78,4 +79,32 @@ def providers
       }
     ]
   end
+
+  def heatmaps
+    # Get latest daily rollup for each node.
+    metrics = MetricRollup.where(:resource_type => ContainerNode.name, :capture_interval_name => "hourly")
+    metrics = metrics.where(:resource_id => @ems.container_nodes) if @ems
+    metrics = metrics.order(:resource_id, :timestamp => :desc)
+    metrics = metrics.select('DISTINCT ON (metric_rollups.resource_id) metric_rollups.*')
+    metrics = metrics.includes(:resource)
```

Comment:
> moving this into the metrics rollup class.
> 
> Is this useful for both metrics and metrics rollups?

@kbrock  Getting the absolute last metrics would be great for realtime metrics but AFAIK we dont really do that in miq.